src/blog/2025/04/building-oee-dashboard-with-flowfuse-2.md

```diff
@@ -0,0 +1,528 @@
+---
+title: "Part 2: Building an OEE Dashboard with FlowFuse"
+subtitle: Step-by-step guide to building your own OEE Dashboard.
+description: Learn how to build an OEE Dashboard using FlowFuse Dashboard (Node-RED Dashboard 2.0). This step-by-step guide covers data collection, OEE calculations, and dashboard creation with simulated production and downtime data.
+date: 2025-04-04
+authors: ["sumit-shinde"]
+image: /blog/2025/04/images/oee-dashboard-buildig-ff-2.png
+keywords: free oee dashboard, oee dashboard free, building oee dashboard, node-red oee dashboard, flowfuse oee dashboard
+tags:
+   - flowfuse
+---
+
+In [Part 1](/blog/2025/04/building-oee-dashboard-with-flowfuse-part-1/), we explored the fundamentals of OEE, outlined a basic design of the dashboard, and identified the key elements to include in the OEE dashboard. 
+
+<!--more-->
+
+In this Part 2, we will focus on building the OEE dashboard interface using [FlowFuse Dashboard](https://dashboard.flowfuse.com/) (Node-RED Dashboard 2.0) and FlowFuse, utilizing simulated production and downtime data.
+
+## Prerequisites
+
+Before you begin building the OEE Dashboard withFlowFuse, make sure you have the following:
+
+- **Running FlowFuse Instance:** Make sure you have a FlowFuse instance set up and running. If you don't have an account, check out our [free trial](https://app.flowfuse.com/account/create) and learn how to create an instance in FlowFuse.
+- **FlowFuse Dashboard:** Ensure you have [FlowFuse Dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) (also known as Node-RED Dashboard 2.0 in the community) installed and properly configured on your instance.
+- **SQLite Contrib Node:** Ensure you have [node-red-contrib-sqlite](https://flows.nodered.org/node/node-red-node-sqlite) installed.
+
+## Getting Started
+
+To simplify the development process, we will divide development into five key parts:
+
+1. Collecting and configuring data
+2. Preparing data for calculations
+3. Calculating OEE and key metrics
+4. Detailed breakdown of OEE data
+5. building the dashboard
```

Comment:
```suggestion
1. Collecting and configuring data
2. Preparing data for calculations
3. Calculating OEE and key metrics
4. Detailed breakdown of OEE data
5. Building the dashboard
```

Additionally to the recommended change, there is an inconsistency between this order and the order of the preceding sections of the article